### ISSUE TITLE
xschema annotations are lax, allowing example without escaping

### ISSUE THREAD

**Issue by crotwell** [FDSN/StationXML#58]
Xschema annotations are lax, allowing any well-formed XML. So there is no need to escape tags within the xschema document. Of course the documatation extractor should run an html char escaper over the text before publishing to html, but that should probably be done anyway. 

For example this:
```
<xs:annotation>
    <xs:documentation>Name of the software module that generated this document.
    </xs:documentation>
    <xs:documentation>Example:&lt;Module&gt;SeisComp3&lt;/Module&gt;</xs:documentation>
</xs:annotation>
```
could be written instead as this, using `<` and `>` which is easier to read when looking at the rawschema:
```
<xs:annotation>
    <xs:documentation>Name of the software module that generated this document.
    </xs:documentation>
    <xs:documentation>Example:<Module>SeisComp3</Module></xs:documentation>
</xs:annotation>
```
Or could even be written with an new tag such as `<example>` which makes the documation extraction processing much easier.
```
<xs:annotation>
    <xs:documentation>Name of the software module that generated this document.
    </xs:documentation>
    <example><Module>SeisComp3</Module></example>
</xs:annotation>
```

This idea could also be used to replace the `Regarding embedded documentation keywords` by simply using attributes on the elements in the annotation. For example:
```
<xs:annotation>
        <xs:documentation>Comment text.</xs:documentation>
          <xs:documentation>LevelChoice:N Example:&lt;Value&gt;Temporary network deployment&lt;/Value&gt;</xs:documentation>
          <xs:documentation>LevelChoice:S Example:&lt;Value&gt;GPS CLock is unlocked&lt;/Value&gt;</xs:documentation>
          <xs:documentation>LevelChoice:C Example:&lt;Value&gt;Large number of spikes&lt;/Value&gt;</xs:documentation>
</xs:annotation>
 ```
could be written as:
```
<xs:annotation>
          <xs:documentation>Comment text.</xs:documentation>
          <xs:documentation LevelChoice="N"><Example><Value>Temporary network deployment</Value></Example></xs:documentation>
          <xs:documentation LevelChoice="S"><Example><Value>GPS CLock is unlocked</Value></Example></xs:documentation>
          <xs:documentation LevelChoice="C"><Example><Value>Large number of spikes</Value></Example></xs:documentation>
</xs:annotation>
```
or even better as
```
<xs:annotation>
          <xs:documentation>Comment text.</xs:documentation>
          <Example LevelChoice="N"><Value>Temporary network deployment</Value></Example>
          <Example LevelChoice="S"><Value>GPS CLock is unlocked</Value></Example>
          <Example LevelChoice="C"><Value>Large number of spikes</Value></Example>
</xs:annotation>
```

Using annotation of this style makes the documentation much easier to read in the raw form, makes editing and maintenance much easier, and is just nicer.

**Comment by crotwell** [comment]
One more thought on this. If the examples are changed to be written as shown, you at least get minimal validation that the example is well formed, which probably catches some typos. 

It would also be possible to either extract the examples from real, validated fdsn stationxml documents, or to inject the examples into a stationxml document in a way that would allow validation of the examples after being written.

**Comment by crotwell** [comment]
Fixed by PR #72

**Comment by jschaeff** [comment]
I did'nt review all the python code but the being able to write actual XML markups in the exemple is an improvement :)
